attachment_sync_gdrive/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Attachment Sync Google Drive
+============================
+
+Automatically upload Account Related attachment  to Google Drive
+
+Installation
+============
+For this module to work you need:
+
+* pip3 install --upgrade google-api-python-client
+* pip3 install --upgrade oauth2client
+
+Configuration
+=============
+
+Add the google drive authorization code
+- *Settings->General Settings->Integrations->Google Drive->Authorization Code*
+
+Credits
+=======
+
+Contributors
+------------
+
+* Dan Kiplangat, Sunflower IT <dan@sunflowerweb.nl>
+* Tom Blauwendraat, Sunflower IT <tom@sunflowerweb.nl>
+
+Images
+------
+
+* Deviant Art: `Icon <https://www.deviantart.com/jasonh1234/art/Google-Drive-Icon-298243611>`_.
```

Comment:
If this is about the credit about who made the image, it's not Deviant Art, but the user: jasonh1234